tests/keras/activations_test.py

```diff
@@ -25,7 +27,32 @@ def test_serialization():
         assert fn == ref_fn
 
 
-def test_softmax():
+def test_get_fn():
+    '''
+    Activations has a convenience "get" function. All paths of this
+    function are tested here, although the behaviour in some instances
+    seems potentially surprising (e.g. situation 3)
```

Comment:
Please format the docstring following conventions used elsewhere in the codebase.